src/Swarm/Game/State.hs

```diff
@@ -144,6 +144,7 @@ import Swarm.Game.Recipe (
   reqRecipeMap,
  )
 import Swarm.Game.Robot
+import Swarm.Game.Scenario.WorldDescription
```

Comment:
Why do we need this new import?

Reply:
Huh, I would have expected a compiler warning about an unused import.  Anyway, I've removed it and it still works.